.github/workflows/coverage.yml

```diff
@@ -0,0 +1,38 @@
+name: Build and Publish
+
+on:
+  push:
+    branches:
+    - 2426-build-on-github-actions-instead-of-appveyor
```

Comment:
I assume the branch name is only for now and will be changed in the future?

Reply:
yeah, I just want to get it working rn. Then we switch to scheduled